### DISCUSSION TITLE
运行报错ID `source/CNAME` has been used

### DISCUSSION THREAD

**Discussion by abctel** [gethexon/hexon#12] [Q&A] [answered]
错误信息：
WarehouseError: ID `source/CNAME` has been used

**Comment by abctel** [comment]
除了该错误，同时还会提醒：Error: ENOENT: no such file or directory, open 'sitemap_template.xml'，应该是文件读取问题，我检测过该文件实际存在。

**Comment by YuJianghao** [comment, marked as answer]
转到 issue 啦
